### PR TITLE
Add MergeObjects module

### PR DESCRIPTION
**Pull request by AetherUnbound**
This resolves the issue in the main CellProfiler repo: 
* https://github.com/CellProfiler/CellProfiler/issues/3381
* https://github.com/CellProfiler/CellProfiler/issues/3369

More discussion around this PR: https://github.com/CellProfiler/CellProfiler/pull/3384

### CODE REVIEW

File: mergeobjects.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# coding=utf-8
+
+"""
+MergeObjects
+===========
+
+**MergeObjects** merges objects below a certain threshold into its most prevalent, adjacent neighbor.
+
+The output of this module is a object image of the same data type as the input.
```
Comment (mcquin) [inline mergeobjects.py]: You could remove this line. It's no longer accurate.

File: mergeobjects.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+**MergeObjects** can be run *after* any labeling or segmentation module (e.g.,
+**ConvertImageToObjects** or **Watershed**). Labels are preserved and, where possible, small
+objects are merged into neighboring objects that constitute a majority of the small object's
+border. This is useful for reversing over-segmentation and artifacts that result from seeding
```
Comment (mcquin) [inline mergeobjects.py]: Would you be interested in contributing a seeding module here? /cc @karhohs

File: tests/test_mergeobjects.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
+import numpy
+import numpy.testing
+import skimage.morphology
+import skimage.segmentation
+import pytest
+import pdb
```
Comment (mcquin) [inline tests/test_mergeobjects.py]: Remove unnecessary import.

File: mergeobjects.py (hunk 2 above)
Comment (AetherUnbound) [reply mergeobjects.py]: Oops I didn't realize this referenced modules that didn't exist in either repo. @karhohs wrote the seeding modules we use I believe, but I can add them to the plugins if he doesn't mind. We've also converted them to `ObjectProcessing` modules from `ImageProcessing` ones.

**Pull request merged**